dotCMS/src/main/java/com/dotmarketing/servlets/ImportExportXMLServlet.java

```diff
@@ -182,6 +179,7 @@ private void doUpload(HttpServletRequest request, HttpServletResponse response)
 		String tempdir = FileUtil.getRealPath(backupTempFilePath);
 
 		MultipartRequest mpr;
+        ZipInputStream zin = null;
 		try {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this "try" to a try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2093)

Reply:
not valid